README.md

```diff
@@ -1,19 +1,22 @@
 [![Open Source Helpers](https://www.codetriage.com/magento/magento2/badges/users.svg)](https://www.codetriage.com/magento/magento2)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/magento/magento2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/magento-2/localized.svg)](https://crowdin.com/project/magento-2)
-<h2>Welcome</h2>
-Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a cutting-edge, feature-rich eCommerce solution that gets results.
+<h2>What is Magento 2 ?</h2>
+It's an e-commerce platform build in PHP, to provide online merchants with a flexible shopping cart system, as well as control over the look, functionality and content of the online store .It also offers powerful search engines for marketing and catalog management tools.
+<br>
```

Comment:
There's no need for these two HMTL line breaks (`<br>`). A single new line is sufficient to separate paragraphs in markdown.